.github/workflows/build.yml

```diff
@@ -0,0 +1,25 @@
+name: Build
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 11
+        uses: olafurpg/setup-scala@v2
+        with:
+          java-version: 11
+      - name: Run tests
+        run: csbt +compile +test scalafmtCheckAll scalastyle
+      - name: Cache Coursier
+        uses: actions/cache@v1
+        with:
+          path: ~/.cache/coursier
+          key: sbt-coursier-cache
+      - name: Cache SBT
+        uses: actions/cache@v1
+        with:
+          path: ~/.sbt
+          key: sbt-${{ hashFiles('**/build.sbt') }}
```

Comment:
You should run the cache actions before the actions that use the cache. This action is actually "restore the cache," and it adds a shutdown hook to upload a new cache.

Also, you can use `cache@v2` and combine multiple directories into a single cache action. @giarc3 has been doing that in our other repos.